stack.yaml

```diff
@@ -1,9 +1,10 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/master/snapshots/cardano-1.24.2.yaml
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/master/snapshots/cardano-1.25.0.yaml
```

Comment:
Should that be `1.25.1` ?

Reply:
Wasn't available when I was starting this, but sure, will test it and change it if it works.